Source/Core/Core/HotkeyManager.h

```diff
@@ -67,6 +67,9 @@ enum Hotkey
   HK_WIIMOTE4_CONNECT,
   HK_BALANCEBOARD_CONNECT,
 
+  HK_NEXT_PROFILE,
```

Comment:
These should likely be `HK_NEXT_CONTROLLER_PROFILE` and `HK_PREV_CONTROLLER_PROFILE` to be explicit.

Reply:
Fixed.